schemas/aws/managed-vpc-1.yml

```diff
@@ -0,0 +1,27 @@
+---
+"$schema": /metaschema-1.json
+version: '1.0'
+type: object
+
+additionalProperties: false
+properties:
+  "$schema":
+    type: string
+    enum:
+    - /aws/managed-vpc-1.yml
```

Comment:
this schema looks a lot like a "request" object similar to what @chassing is doing with AWS account requests.

i'm not sure about your plan but to me the flow of events to create a VPC could be as follows:
* create a `/aws/managed-vpc-1.yml` 
* an integration picks it up, creates a VPC in the account
* the integration then creates an MR to introduce a datafile of schema `/aws/vpc-1.yml` with all the details like VPC IDs
* we keep using `/aws/vpc-1.yml` in all kinds of places where we need to reference VPCs

was that the plan?

Reply:
That was not the plan at that point. But now it is. Thanks for the insight. I'm going to rename things to make this more clear.